mu4e-dashboard.el

```diff
@@ -42,7 +42,18 @@
 (defvar mu4e-dashboard--buffer nil)
 
 ;; Install the mu4e link type
-(org-add-link-type "mu4e" 'mu4e-dashboard-follow-mu4e-link)
+(defgroup mu4e-dashboard nil
+  "Provides a new Org mode link type for mu4e queries."
+  :group 'comm)
+
+(defcustom mu4e-dashboard-link-name "mu4e"
+  "Default link name."
+  :group 'mu4e-dashboard
```

Comment:
You don't really have to repeat the `:group` here, as it will default to the last `defgroup` in the file.  It's a detail, but saves a bit of typing.

My 0.02 cents :)

Reply:
Huh-- I didn't know that. Thanks!

> If a ‘defcustom’ does not specify any ‘:group’, the last group defined with ‘defgroup’ in the same file will be used.  This way, most ‘defcustom’ do not need an explicit ‘:group’.